src/agent/web.ts

```diff
@@ -416,4 +417,34 @@ export default class Web extends WebAPI {
   async listAgentsForTransfer(chat_id: string): Promise<AgentForTransfer[]> {
     return this.send("list_agents_for_transfer", { chat_id });
   }
+
+  /**
+   * Marks a customer as followed. As a result, the requester (an agent) will receive
+   * the info about all the changes related to that customer via pushes.
+   * @param id - ID of customer
+   */
+  async followCustomer(id: string): Promise<EmptyResponse> {
+    return this.send("follow_customer", { id });
+  }
+
+  /**
+   * Removes the agent from the list of customer's followers.
+   * @param id - ID of customer
+   */
+  async unfollowCustomer(id: string): Promise<EmptyResponse> {
+    return this.send("unfollow_customer", { id });
+  }
+
+  /**
+   * Returns the current routing status of each agent.
+   * @param groupIds - Array of group IDs.
+   */
+  async listRoutingStatuses(groupIds?: number[]): Promise<SetRoutingStatusResponse[]> {
```

Comment:
i think we use snake_case to name params, check other similar places for reference

Reply:
That's right, changed.

Reply:
yup, in most places we use function param names to directly construct req payloads, which require snake_case, therefore snake_case is chosen here (camelCase everywhere else)